sources/packages/backend/apps/api/src/route-controllers/confirmation-of-enrollment/confirmation-of-enrollment.institutions.controller.ts

```diff
@@ -234,13 +243,14 @@ export class ConfirmationOfEnrollmentInstitutionsController extends BaseControll
    */
   @HasLocationAccess("locationId")
   @ApiNotFoundResponse({
-    description:
-      "Confirmation of enrollment not found or application status not valid.",
+    description: "Enrolment not found.",
   })
   @ApiUnprocessableEntityResponse({
     description:
-      "Tuition amount provided should be lesser than both (actual tuition + program related costs) and (Canada grants + Canada Loan + BC Loan) " +
-      `or confirmation of enrollment window is greater than ${COE_WINDOW} days ` +
+      "Enrolment already completed and can neither be confirmed nor declined" +
```

Comment:
I would preferer to have it adjusted to look like a proper sentence on Swagger.
For instance, adding spaces at the end, do not add periods if the next part starts with an or and upper case.